docs/docs/forms.mdx

```diff
@@ -360,74 +360,101 @@ to fully customize the validation process.
 
 If none of the predefined [Slot Mappings](./forms.mdx#slot-mappings) fit your use
 case, you can use the
-[Custom Action](./actions.mdx#custom-actions) `validate_{form_name}` to write your own
-extraction code. Rasa Open Source will trigger this function when the form is run.
+[Custom Action](./actions.mdx#custom-actions) `validate_<form_name>` to write your own
+extraction code. Rasa Open Source will trigger this action when the form is run.
 
-Make sure your custom action returns `SlotSet` events for every extracted value.
-The following example shows the implementation of a custom slot mapping which sets
-a slot based on the length of the last user message.
+If you're using the Rasa SDK we recommend you to extend the provided
+`FormValidationAction`. When using the `FormValidationAction`, two things are needed to
+extract customs slots:
+
+1. Define a method `extract_<slot_name>` for every custom slot mapping which you want to
+   extract.
```

Comment:
What about this:
```suggestion
1. Define a method `extract_<slot_name>` for every slot that should be extracted using a custom mapping.
```
My motivation is to get rid of the `custom slot` term as much as possible to stress out that it is the mapping that is `custom`. Though the possible confusion between `extracting` and `mapping` remains... Then perhaps this could partly do the job?
```suggestion
1. Define a method `extract_<slot_name>` for every slot that should be mapped in a custom way.
```